sphinx/pycode/ast.py

```diff
@@ -141,7 +141,7 @@ def visit_Constant(self, node: ast.Constant) -> str:
     def visit_Dict(self, node: ast.Dict) -> str:
         keys = (self.visit(k) for k in node.keys if k is not None)
         values = (self.visit(v) for v in node.values)
-        items = (k + ": " + v for k, v in zip(keys, values))
+        items = (k + ": " + v for k, v in zip(keys, values, strict=False))
```

Comment:
Just wondering but wouldn't it be inconsistent if strict == False? because otherwise, the key-value pair would not be related...